controller/app/index.js

```diff
@@ -0,0 +1,93 @@
+import "./jquery-global";
+import "bootstrap-cron-picker/dist/cron-picker";
+
+$().ready(() => {
+    $('#newSchedule').cronPicker();
+
+    $("#addDone").hide();
+    if ($('#newSR').is(':checked')) {
+        $("#newstorage").hide();
+    } else {
+        $("#newretrieval").hide();
+    }
+    $("#newSR").on('change', () => {
+        if ($('#newSR').is(':checked')) {
+            $("#newretrieval").show();
+            $("#newstorage").hide();
+        } else {
+            $("#newretrieval").hide();
+            $("#newstorage").show();
+        } 
+    })
+
+    if (!$('#newRepeat').is(':checked')) {
+        $("#setschedule").hide();
+    }
+    $("#newRepeat").on('change', () => {
+        if ($('#newRepeat').is(':checked')) {
+            $("#setschedule").show();
+        } else {
+            $("#setschedule").hide();
```

Comment:
isn't there a "toggle show/hide" method, or something like "set hidden as per this boolean"?

Reply:
possibly?